test/minikube/minikube_long_admin_test.go

```diff
@@ -14,7 +14,11 @@ func TestKubernetesBasicAdminThreeReplicas(t *testing.T) {
 	testlib.AwaitTillerUp(t)
 
 	options := helm.Options{
-		SetValues: map[string]string{"admin.replicas": "3"},
+		SetValues: map[string]string{
+			"admin.replicas": "3",
+			"admin.k8sL4Service.enabled": "true",
+			"admin.k8sL4Service.internalIP": "false",
```

Comment:
is this something leftover from testing? Why is it required? All these tests are being run in minikube, not in AWS/Google/Azure. There is no L4Service

Reply:
Without it the existing test fails - minikube creates the service but it never gets it's external IP address (since there is no cloud).  So it's either remove the test or add the new values so it still creates in the same way, and the test passes...

Reply:
Got it. I want to make sure that the default properties work and give you a functional environment.

Reply:
As discussed, removed all minikube L4 tests